src/win32/win_agent.c

```diff
@@ -20,6 +20,7 @@
 #include "os_execd/execd.h"
 #include "os_crypto/md5/md5_op.h"
 #include "external/cJSON/cJSON.h"
+#include "dll_load_notify.h"
```

Comment:
Why is this added?

Reply:
No falta el llamado a "enable_dll_verification()" en el main?

Reply:
In some previous commits, I had added the `enable_dll_verification()` here. But this executable is used to enable the service also and the verification method caused a delay that prevented Windows to properly start the Wazuh agent service.

The `enable_dll_verification()` is placed in `local_start()` for the `wazuh-agent`.